nav2_amcl/package.xml

```diff
@@ -15,25 +15,22 @@
       'amcl' Player driver.
     </p>
   </description>
-  <url>http://wiki.ros.org/amcl</url>
-  <author>Brian P. Gerkey</author>
```

Comment:
I agree and my vote is to keep the author field, but per our discussion we decided to remove author fields.

Reply:
We'd agreed at the beginning of the project to leave out the author fields in the code and .xml files. The trouble is who do you count as an author? Technically, everyone who contributed a change to the package could be an author.

The list would probably be twenty people long at least.

If you don't count everyone who contributed a change, where do you draw the line to cut them off?

My preference is to not list authors at all to avoid this problem.

Reply:
I think that's fair for packages that have been created in this repo, but for packages ported over from navigation, I think we should at least ask these folks like (@gerkey in this instance) if he's cool with that

Reply:
@mjeronimo @crdelsey How about keeping the name of the main author for the packages that are ported over?

@SteveMacenski amcl_node.cpp still has the main author's name.